packages/frontend/src/creation-form/payload-form/tvl.tsx

```diff
@@ -25,27 +30,35 @@ export const TvlPayloadForm = ({ t, payload, onChange }: PayloadFormProps) => {
     }, [onChange, protocol]);
 
     return (
-        <div className="flex flex-col gap-2 md:flex-row">
-            <div className="w-full">
-                <Select
-                    id="protocol"
-                    className={{
-                        root: "w-full",
-                        input: "w-full",
-                        inputWrapper: "w-full",
-                    }}
-                    label={t("label.tvl.protocol")}
-                    placeholder={t("placeholder.pick.protocol")}
-                    loading={loading}
-                    messages={{
-                        noResults: t("select.no.results"),
-                    }}
-                    onChange={setProtocol}
-                    options={protocols}
-                    search
-                    value={protocol}
-                />
+        <div className="flex flex-col gap-4">
+            <div className="flex flex-col gap-2 md:flex-row">
+                <div className="w-full">
+                    <Select
+                        id="protocol"
+                        className={{
+                            root: "w-full",
+                            input: "w-full",
+                            inputWrapper: "w-full",
+                        }}
+                        label={t("label.tvl.protocol")}
+                        placeholder={t("placeholder.pick.protocol")}
+                        loading={loading}
+                        messages={{
+                            noResults: t("select.no.results"),
+                        }}
+                        onChange={setProtocol}
+                        options={protocols}
+                        search
+                        value={protocol}
+                    />
+                </div>
             </div>
+            {measurementTimestamp && protocol && (
+                <Typography>
+                    Metric summary: total value locked in {protocol.label} on
+                    date {measurementTimestamp.format("L HH:mm:ss")}
```

Comment:
Is `L` a typo? It's not supported by [dayjs](https://day.js.org/docs/en/display/format).